conversion/conversion.go

```diff
@@ -0,0 +1,670 @@
+package conversion
```

Comment:
Every package should have a comment of the form:

// Package conversion ... <description of what it does>.

Note that only one file per package should have comment -- this seems like the main file for the package?

Reply:
added comments on top of web.go in web and conversion.go in conversion.